MacPass/MPPasswordInputController.m

```diff
@@ -177,4 +182,48 @@ - (void)_showError:(NSError *)error {
   self.messageInfoTextField.hidden = NO;
 }
 
+
+- (NSTouchBar *)makeTouchBar {
+  NSTouchBar *touchBar = [[NSTouchBar alloc] init];
+  touchBar.delegate = self;
+  touchBar.defaultItemIdentifiers = @[touchBarShowPasswordIdentifier, touchBarChooseKeyfileIdentifier, NSTouchBarItemIdentifierFlexibleSpace,touchBarUnlockIdentifier];
+  return touchBar;
+}
+
+- (NSTouchBarItem *)touchBar:(NSTouchBar *)touchBar makeItemForIdentifier:(NSTouchBarItemIdentifier)identifier  API_AVAILABLE(macos(10.12.2)) {
+  if (identifier == touchBarChooseKeyfileIdentifier) {
+    NSCustomTouchBarItem *item = [[NSCustomTouchBarItem alloc] initWithIdentifier:touchBarChooseKeyfileIdentifier];
+    NSImage *image = [NSImage imageNamed:NSImageNameTouchBarFolderTemplate];
+    NSButton *button = [NSButton buttonWithTitle:@"Choose Keyfile" image:image target:self.keyPathControl action:@selector(showOpenPanel:)];
+    item.view = button;
+    return item;
+  } else if (identifier == touchBarShowPasswordIdentifier) {
+    NSCustomTouchBarItem *item = [[NSCustomTouchBarItem alloc] initWithIdentifier:touchBarShowPasswordIdentifier];
+    NSImage *image = [NSImage imageNamed:NSImageNameTouchBarQuickLookTemplate];
+    NSButton *button = [NSButton buttonWithTitle:@"Show Password" image:image target:self action:@selector(toggleShowPassword)];
+    // TODO: Set active when toggled with
+    // [button highlight:true];
+    item.view = button;
+    return item;
+  } else if (identifier == touchBarUnlockIdentifier) {
+    NSCustomTouchBarItem *item = [[NSCustomTouchBarItem alloc] initWithIdentifier:touchBarUnlockIdentifier];
+    NSImage *image = [NSImage imageNamed:NSImageNameLockUnlockedTemplate];
+    NSButton *button = [NSButton buttonWithImage:image target:self action:@selector(_submit:)];
+    // TODO: update color if the appearence changes
+    if (@available(macOS 10.14, *)) {
+      [button setBezelColor:NSColor.controlAccentColor];
```

Comment:
This is a property, so just use it that way.
button.bezelColor = NSColor.contolAccentColor;